resteasy-cdi/src/main/java/org/jboss/resteasy/cdi/ResteasyCdiExtension.java

```diff
@@ -78,6 +85,47 @@ public void observeBeforeBeanDiscovery(@Observes BeforeBeanDiscovery event, Bean
       active = true;
    }
 
+   public <T> void observeStereotypes(@WithAnnotations({Stereotype.class}) @Observes ProcessAnnotatedType<T> event, BeanManager manager)
+   {
+      AnnotatedType<T> annotatedType = event.getAnnotatedType();
+      Set<Annotation> jaxRsAnnotations = collectJaxRsAnnotation(annotatedType.getAnnotations(), beanManager);
+
+      //TODO fix isJaxrsAnnotatedClass?
```

Comment:
I'm not sure about the details, but this probably needs to be clarified before merging, right?

Reply:
sorry, I missed this in cleanup